modules/bank.js

```diff
@@ -303,7 +305,7 @@ async function slashBankDiscount(interaction) {
         interaction.followUp("I wasn't able to send you the code! Do you have DMs allowed for server members? Please check with a member of Management to get your discount code.");
       });
       const embed = u.embed()
-      .setAuthor(interaction.member.displayName, interaction.member.user.displayAvatarURL({ dynamic: true }))
+      .setAuthor({ name: interaction.member.displayName, iconURL: interaction.member.user.displayAvatarURL({ dynamic: true }) })
```

Comment:
This is one of the few places that did need updating. Keep the change you made here (or even better, move the author into the call to u.embed() as is done in most other places).